server/service/src/apis/api_on_central/mod.rs

```diff
@@ -0,0 +1,122 @@
+use log::info;
```

Comment:
I'm not quite sure where this file should live? Didn't belong with sync stuff, maybe here is fine?